modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -312,10 +312,12 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
       const formatItem = formatList[i];
       if (formatItem.ceFalseOverride && FormatUtils.isSelectorFormat(formatItem) && dom.is(node, formatItem.selector)) {
         setElementFormat(node, formatItem);
+        ed.fire('FormatApply', { format: name, vars });
```

Comment:
All `fire` calls should be in the `Events.ts` file, so please move these there.

- [ ] Move all `ed.fire` calls to `Events.ts`

Reply:
Additionally this duplicates the fire call, so this could be improved by removing this and just changing the below line to be `break` instead.